### PR TITLE
Use new db

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-1055

### Change description ###

new db created #299,
now changing the connection configuration .
in next pr will remove obsolete configuration

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/reform-scan-blob-router/values.yaml [hunk 1]
```diff
@@ -14,17 +14,17 @@ java:
...
     DB_PORT: '5432'
     DB_NAME: blob_router
-    DB_USER: blob_router@reform-scan-blob-router-{{ .Values.global.environment }}
+    DB_USER: blob_router@blob-router-{{ .Values.global.environment }}
     DB_CONN_OPTIONS: ?sslmode=require
 
-    FLYWAY_URL: jdbc:postgresql://reform-scan-blob-router-{{ .Values.global.environment }}.postgres.database.azure.com:5432/blob_router?sslmode=require
-    FLYWAY_USER: blob_router@reform-scan-blob-router-{{ .Values.global.environment }}
+    FLYWAY_URL: jdbc:postgresql://blob-router-{{ .Values.global.environment }}.postgres.database.azure.com:5432/blob_router?sslmode=require
+    FLYWAY_USER: blob_router@blob-router-{{ .Values.global.environment }}
```
Comment (doncem) [inline charts/reform-scan-blob-router/values.yaml]: if "fixing" the above then need to clarify how those can be plugged in so the db step in pipeline can execute. spring application can be derived from existing config but not sure about db migration step :thinking:

File: charts/reform-scan-blob-router/values.yaml [hunk 2]
```diff
@@ -14,17 +14,17 @@ java:
...
-        - blob-router-POSTGRES-PASS
+        - db-flyway-password
+        - blob-router-db-pass
   environment:
-    DB_HOST: reform-scan-blob-router-{{ .Values.global.environment }}.postgres.database.azure.com
+    DB_HOST: blob-router-{{ .Values.global.environment }}.postgres.database.azure.com
     DB_PORT: '5432'
     DB_NAME: blob_router
-    DB_USER: blob_router@reform-scan-blob-router-{{ .Values.global.environment }}
+    DB_USER: blob_router@blob-router-{{ .Values.global.environment }}
```
[the remark this answers is not in the document]
Comment (mdayican) [reply charts/reform-scan-blob-router/values.yaml]: not now, I do not want mix that with upgrade.

**Comment by mdayican** [comment]
> Can you please use appropriate commit messages ?

this commits happened because github was down.

File: charts/reform-scan-blob-router/values.yaml (hunk 1 above)
Comment (mdayican) [reply charts/reform-scan-blob-router/values.yaml]: I think we need code change,I do not want mix that with upgrade.

**Comment by mdayican** [comment]
as requested

**Pull request closed without merging**